src/app/fdctl/run/tiles/fd_replay.c

```diff
@@ -292,6 +298,17 @@ during_frag( void * _ctx,
     }
 
     FD_LOG_INFO(( "packed microblock - slot: %lu, parent_slot: %lu, txn_cnt: %lu", ctx->curr_slot, ctx->parent_slot, ctx->txn_cnt ));
+  } else if( in_idx == GOSSIP_IN_IDX ) {
+    if( FD_UNLIKELY( chunk<ctx->gossip_in_chunk0 || chunk>ctx->gossip_in_wmark || sz>USHORT_MAX ) ) {
+      FD_LOG_ERR(( "chunk %lu %lu corrupt, not in range [%lu,%lu]", chunk, sz, ctx->gossip_in_chunk0, ctx->gossip_in_wmark ));
+    }
+    uchar * src = (uchar *)fd_chunk_to_laddr( ctx->gossip_in_mem, chunk );
+    /* Incoming packet from gossip tile, a raw vote transaction.
+     */
+    fd_compact_vote_state_update_t vote;
+    FD_TEST( FD_VOTE_TXN_PARSE_OK == fd_vote_txn_parse(src, sz, ctx->replay->valloc, &vote) );
+    FD_LOG_WARNING(( "Yunhao: Replay tile succeeds in parsing vote txn size=%u, vote root=%lu", sz, vote.root ));
+    return;
```

Comment:
what you really want to do is memcpy this txn into a buffer in ctx for a single vote. then you can parse the buffer in after_frag